proto/authzed/api/validation_test/tuples_test.go

```diff
@@ -54,11 +54,12 @@ var objectIDs = []struct {
 	{"A-A-A", true},
 	{"123e4567-e89b-12d3-a456-426614174000", true},
 	{strings.Repeat("f", 64), true},
+	{strings.Repeat("f", 128), true},
 	{"", false},
 	{"  ", false},
 	{"-", false},
 	{strings.Repeat("\u0394", 4), false},
-	{strings.Repeat("f", 65), false},
+	{strings.Repeat("f", 129), false},
```

Comment:
maybe we should make 64 a constant, same with 128, and then this test can read max+1